ansible/roles/common/tasks/redhat.yml

```diff
@@ -18,3 +18,16 @@
   yum:
     name: "{{ item }}"
   with_items: "{{ common_extra_rpms }}"
+
+- name: Load br_netfilter kernel module
+  modprobe:
+    name: br_netfilter
+    state: present
+
+- name: Set sys.net.bridge.bridge-nf-call-iptables
+  sysctl:
+    name: net.bridge.bridge-nf-call-iptables
+    value: 1
+    sysctl_set: yes
+    state: present
+    reload: yes
```

Comment:
It looks like we don't currently do this on Debian, but we do set the sysctl in ansible/roles/kubernetes-cni/flannel.yml.

I'm wondering if it would make sense to just set it for both RH and Debian here in the `common` role? @craigtracey thoughts?

Reply:
I think Craig may be out-of-pocket in Switzerland. I'll build a Wardroom-prepped Ubuntu 16.04 image and see if I can reproduce the error there.

Reply:
@detiber I just tested Wardroom with Ubuntu 16.04 and K8s 1.11.2. The same error was I seeing on CentOS (regarding /proc/sys/net/bridge/bridge-nf-call-iptables) did not come up at all. This implies that we don't need to set this value for Ubuntu/Debian.